### DISCUSSION TITLE
Contributor number inaccurate

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#19826] [Pull Requests] [answered]
I have created an PR and have seen that's written there, that there are two contributors, but i am only one:
![grafik](https://user-images.githubusercontent.com/50182241/176897936-e6dfb634-e167-44ba-9fcb-685732e08fc5.png)

I think this is because there are contributions from my pc and my laptop.
The only thing i can think of is, that it is because they don't share the same GPG Key

**Comment by themiralay** [comment]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!
 
 Other (possible) duplicates are:
 
 * [[Projects Beta] Show number of tasks vs completed task after issue title #16616](https://github.com/orgs/github-community/discussions/16616)
 * [[Projects Beta] Show x of n tasks on issue #13033](https://github.com/orgs/github-community/discussions/13033)
 * [[Projects beta] Task list progress #16721](https://github.com/orgs/github-community/discussions/16721)
 * [Show percent tasks complete in a new projects Beta field #12050](https://github.com/orgs/github-community/discussions/12050)

**Reply by CuzImBisonratte** [reply]
All of those have nothing to do with my issue.  
The ones you named are all related to issue task progress.  

My issue has to do with the number of contributors to a PR.

**Comment by manasesjesus** [comment, marked as answer]
Do you have several emails in your settings? GitHub displays the username and hides emails, so commits with different emails can be one reason. Using different tools (web, CLI, client) to commit and merge could be another reason. My guess is that your **Merge pull request 108** commit is counted as the second contributor.

Check your commits history in the terminal and see if the `Author: your name <your@email.com>` is the same in all commits.
```
$ git log
```

As a reference, when I use web, CLI and the desktop app, the history shows different names. 

<img width="309" alt="commits" src="https://user-images.githubusercontent.com/24204142/177211337-757aa7e8-00fd-44ef-824e-71293ee07b27.png">

**Reply by CuzImBisonratte** [reply]
Well, you are right, once i used my normal mail and once i used my mail without the capital letters:
![grafik](https://user-images.githubusercontent.com/50182241/177328961-89164405-292e-4ce8-9688-8e88048aff78.png)
